index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <header id="header">
+      <nav id="nav-bar">
+        <ul>
+      
+      <li><a class="nav-link" href="#about">About</a></li>
+      <li><a class="nav-link"  href="#contact">Contact</a></li>
+      <li><a class="nav-link" href="#skills">skills</a></li>
+      </ul>
+      </nav>
+      <img id="header-img" src="https://www.bing.com/th?id=OIP.FcUYoInKYogVky8OJn08lgHaLH&w=120&h=170&c=8&rs
+      =1&qlt=90&o=6&pid=3.1&rm=2" alt="nature image"
+    </header>
+    <!--about section-->
+    <section id="about">
+      
+      <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Magnam dolorem 
+        voluptatum dolorum consectetur amet eos voluptatibus obcaecati? 
+        Earum, beatae amet. Officia, eveniet? Quo, amet. Explicabo.</p>
+    </section>
+
+    <!--contact section-->
```

Comment:
Please avoid the comment contact section in your index.html file because it is unused code